libraries/AP_UAVCAN/AP_UAVCAN.h

```diff
@@ -124,7 +124,7 @@ class AP_UAVCAN : public AP_HAL::CANProtocol {
             return uavcan::UtcTime::fromUSec(AP_HAL::micros64() + utc_adjustment_usec);
         }
 
-        static SystemClock& instance() {
+        static SystemClock& get_singleton() {
```

Comment:
I think this one is defined in uavcan, so needs to stay as instance()

Reply:
yeah, I was actually on the fence about that one. I've deleted that commit and force pushed